src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -359,6 +332,45 @@ void DnssdService::ToDiscoveredNodeData(const Span<Inet::IPAddress> & addresses,
     }
 }
 
+CHIP_ERROR DnssdService::ToResolvedNodeData(const Span<Inet::IPAddress> & addresses, ResolvedNodeData & nodeData)
+{
+    auto & resolutionData  = nodeData.resolutionData;
+    auto & operationalData = nodeData.operationalData;
+
+    PeerId peerId;
+    ReturnErrorOnFailure(ExtractIdFromInstanceName(mName, &peerId));
+
+    Platform::CopyString(resolutionData.hostName, mHostName);
+    IPAddressSorter::Sort(addresses, mInterface);
```

Comment:
I did not see this Sort() on the old code.  Is this being added with this change?

Reply:
That is me beeing overzealous... I have removed that.